### PR TITLE
libiio: fix cross compilation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by amjoseph-nixpkgs** [comment]
@ofborg build pkgsCross.aarch64-multiplatform.libiio

File: pkgs/development/libraries/libiio/default.nix [hunk 1]
```diff
@@ -50,6 +52,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DUDEV_RULES_INSTALL_DIR=${placeholder "out"}/lib/udev/rules.d"
+    "-DPython_EXECUTABLE=${buildPackages.python3}/bin/python"
```
Comment (Artturin) [inline pkgs/development/libraries/libiio/default.nix:55]:
```suggestion
    "-DPython_EXECUTABLE=${python.pythonForBuild.interpreter}"
```

so there's no version mismatch

File: pkgs/development/libraries/libiio/default.nix [hunk 2]
```diff
@@ -37,6 +38,7 @@ stdenv.mkDerivation rec {
     flex
     bison
     pkg-config
+    python
   ];
 
   buildInputs = [
```
Comment (Artturin) [inline pkgs/development/libraries/libiio/default.nix:44]: Perhaps the python stuff can be removed from buildInputs

Comment (Artturin) [reply pkgs/development/libraries/libiio/default.nix:43]: diffed and yep it can be

**Pull request merged**